packages/tasks/src/model-libraries.ts

```diff
@@ -553,6 +553,14 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: true,
 		countDownloads: `path_extension:"sentis"`,
 	},
+	"videocrafter": {
+		prettyLabel: "VideoCrafter",
+		repoName: "VideoCrafter",
+		repoUrl: "https://github.com/AILab-CVC/VideoCrafter",
+		snippets: snippets.videocrafter,
+		filter: true,
```

Comment:
```suggestion
```

`filter: true` would list the library in the main libraries on hf.co/models, which we don't necessarily want to do